site/app/AppKernel.php

```diff
@@ -27,6 +27,7 @@ public function registerBundles()
             new Exercise\HTMLPurifierBundle\ExerciseHTMLPurifierBundle(),
             new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
             new Sentry\SentryBundle\SentryBundle(),
+            new Http\HttplugBundle\HttplugBundle(),
```

Comment:
Where do we use this?

Reply:
It's a dependency of the updated HWIOAuthBundle